core/src/main/java/org/apache/accumulo/core/client/mapred/AbstractInputFormat.java

```diff
@@ -484,7 +509,7 @@ public void initialize(InputSplit inSplit, JobConf job) throws IOException {
       if (null == authorizations) {
         authorizations = getScanAuthorizations(job);
       }
-
+      String classLoaderContext = getContext(job);
```

Comment:
None of the other variables in the method have the final modifier. Are you trying to raise the bar?